src/pokete/classes/fight/fight.py

```
@@ -187,31 +196,46 @@ def __call__(self, ctx: Context, providers: list[Provider]):
+            'faints': player_faints,
+            'final_hp_ratio': final_hp_ratio,
+            'unique_types': len(player_types_used),
+            'player_won': player_won,
+            'trainer_name': trainer_name
+        }
+
+        dm.report_battle(telemetry)
+        dm.update_team_analysis(ctx.figure)
+
+        if player_won:
             winner.curr.poke_stats.add_battle(True)
```

Ruff (0.14.14)

[error] Boolean positional value in function call

(FBT003)